packages/frontend/app/package.json

```diff
@@ -41,7 +41,16 @@
   },
   "devDependencies": {
     "@types/enzyme-adapter-react-16": "^1.0.5",
+    "husky": "^3.0.7",
     "jest": "^24.9.0",
-    "ts-jest": "^24.1.0"
+    "prettier": "^1.18.2",
+    "pretty-quick": "^1.11.1",
+    "ts-jest": "^24.1.0",
+    "tslint-config-prettier": "^1.18.0"
```

Comment:
`tslint` will be deprecated in favour of `eslint`, please consider moving to `eslint` 
https://www.npmjs.com/package/tslint